docs/userguide/expectations.md

```diff
@@ -14,66 +14,213 @@ Expectation is a set of the expected value(s), actual values(s) and the matcher(
 
 Matcher is defining the comparison operation to be performed on expected and actual values. 
 
-# List of currently build-in matchers
-- `match`
-- `equal`
-- `be_true`
-- `be_null`
-- `be_not_null`
-- `be_like`
-- `be_less_than`
-- `be_less_or_equal`
-- `be_greater_than`
-- `be_greater_or_equal`
-- `be_false`
+# Matchers
```

Comment:
I didn't mention the shortcuts. We could add them as a separate section.
I think some shortcuts are not complete (not all data types have shortcuts to supported matchers). This is why they re not mentioned in the documentation.